frontend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:20-alpine AS build
+WORKDIR /app
+RUN apk add --no-cache python3 make g++
+COPY package*.json ./
+RUN npm ci
+COPY . .
+RUN npm run build
+
+FROM node:20-alpine
+WORKDIR /app
+ENV NODE_ENV=production
+COPY --from=build /app ./
```

Comment:
and you are sure this works? because I had to add an adapter for node (node dist/server/entry.mjs)

Reply:
yes